app/javascript/components/enrollment/Enrollment.js

```diff
@@ -135,7 +138,7 @@ class Enrollment extends React.Component {
       .then(response => {
         if (response.data && response.data.is_duplicate) {
           const dupFieldData = response.data.duplicate_field_data;
-          const patientType = this.state.enrollmentState.isolation ? 'case' : 'monitoree';
+          const patientType = this.state.current.patient.isolation ? 'case' : 'monitoree';
```

Comment:
where is `state.current` defined in this component?

Reply:
this was a typo: https://github.com/SaraAlert/SaraAlert/pull/1027/commits/7996bd8476129821796c62f818be44c70f49f29a